library/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@
 
 
 # This is incremented when the ABI to the library changes
-set( hipsolver_SOVERSION 0.1 )
+set( hipsolver_SOVERSION 0.2 )
```

Comment:
Can we update the SO version in this PR to 0.3?

Reply:
Sure. I also started escaping the regexes in `bump_hipsolver_version.sh`, as I noticed that `sed 's/0.4/0.5/g'` would change '(C) 2024' to '(C) 2025'.